CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Unreleased changes
 
-[Full changelog](https://github.com/mozilla/glean/compare/v22.0.0...master)
+[Full changelog](https://github.com/mozilla/glean/compare/v22.1.0...master)
+
+# v22.1.0 (2019-12-17)
+
+[Full changelog](https://github.com/mozilla/glean/compare/v22.0.0...22.1.0)
```

Comment:
```suggestion
[Full changelog](https://github.com/mozilla/glean/compare/v22.0.0...v22.1.0)
```

I fixed this in master already, so this is essentially backporting it.

Reply:
Got it, makes sense!